frontend/src/metabase/styled-components/selectors.ts

```diff
@@ -7,8 +7,8 @@ import { getSettings } from "metabase/selectors/settings";
 export const getFont = createSelector(
   [getSettings, getEmbedOptions],
   (settings, embedOptions) => {
-    if (embedOptions.font) {
-      return embedOptions.font;
+    if (embedOptions?.font) {
```

Comment:
Why do we have a situation where `embedOptions` is nully? I see that we have `embed` in our SDK as well as you could see here.

![image](https://github.com/user-attachments/assets/204131f6-f5df-4035-b60b-1eb82a3c58fe)

The `embed.options` also has a default value of `{}`
https://github.com/metabase/metabase/blob/e9db88131ff744c72309cd259775e642783b0066/frontend/src/metabase/redux/embed.ts#L36-L39

Reply:
Good point, it was failing before, I just pushed a commit that should both explain why and fix the underlaying issue :)

Reply:
Ah cool.